coordinator/tests/services/translation/request/prescription.spec.ts

```diff
@@ -344,21 +344,25 @@ function setReviewDate(medicationRequest: fhir.MedicationRequest, newReviewDate:
   reviewDateExtension.valueDateTime = newReviewDate
 }
 
-function clearRepeatInformation(medicationRequest: fhir.MedicationRequest) {
+function clearRepeatInformationExtension(medicationRequest: fhir.MedicationRequest) {
   const repeatInformationExtension = getExtensionForUrl(
     medicationRequest.extension,
     "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-MedicationRepeatInformation",
     "MedicationRequest.extension"
-  ) as fhir.RepeatInformationExtension
+  ) as fhir.UkCoreRepeatInformationExtension
   medicationRequest.extension.splice(medicationRequest.extension.indexOf(repeatInformationExtension), 1)
 }
 
+function clearRepeatInformationNumberOfRepeatsAllowed(medicationRequest: fhir.MedicationRequest) {
+  medicationRequest.dispenseRequest.numberOfRepeatsAllowed = ""
```

Comment:
Maybe `delete` the field instead of setting it to empty string. This field is technically not allowed to be a string (although we do handle it just in case)